### DISCUSSION TITLE
Compilation error

### DISCUSSION THREAD

**Discussion by Kamaxo93** [Tlaster/PreCompose#357] [Q&A]
Good morning, everyone,
when updating the library to version 1.6.1 I get the following error:FIELD name:moe_tlaster_precompose_navigation_Navigation_Navigator$stable type:kotlin.Int visibility:public [final,static]

**Comment by Tlaster** [comment]
I'm sorry for the recent inactivity in this repository. I've been in poor health and quite busy lately. I hope to provide some updates to this repo later this week.
